src/router/kromo/dd-wrt/lang_pack/english.js

```diff
@@ -799,9 +801,9 @@ hfirewall.page7="<dd>Prevents hosts on LAN from using WAN address of router to c
 hfirewall.page8="<dd>Prevents WAN access to port 113.</dd>";
 hfirewall.page9="<dd>The router can keep logs of all incoming or outgoing traffic for your Internet connection.</dd>";
 hfirewall.page10="<dd>To keep activity logs, select <em>" + share.enable + "</em>. To stop logging, select <em>" + share.disable + "</em>.</dd>";
-hfirewall.page11="<dd>Set this to the required amount of information. Set <i>Log Level</i> higher to log more actions.</dd>";
-hfirewall.page12="<dd>To see a temporary log of the router's most recent incoming traffic, click the <i>Incoming Log</i> button.</td>";
-hfirewall.page13="<dd>To see a temporary log of the router's most recent outgoing traffic, click the <i>Outgoing Log</i> button.</dd><dd>Check all values and click the <em>" + sbutton.save + "</em> button to save your settings. Click the <em>" + sbutton.cancel + "</em> button to cancel your unsaved changes.</dd>";
+hfirewall.page11="<dd>Set this to the required amount of information. Set <em>" + log.lvl + "</em> higher to log more actions.</dd>";
```

Comment:
`log.lvl` is being called before defined defined.

![Capture](https://user-images.githubusercontent.com/31389848/179392050-0e018af0-e1f5-4d9d-b068-acd7cebc51c5.PNG)

![Capture](https://user-images.githubusercontent.com/31389848/179392083-fd39a8d1-4135-4be4-b8e4-b240a697c7e8.PNG)

Reply:
and that does not work?
that is a garbage

I'll revise it... but will not happen in the next 1-2h

Reply:
I already fixed it.

Its javascript, you cant call a variable before its defined it wont work and its not rubbish ;)

Reply:
It's garbage since now I have more to check.
The text walls are already very difficult to translate. (all in one line)

I have checked my changes. 
It is actually only the string "log.lvl".

hotfix > https://github.com/mirror/dd-wrt/pull/280

Reply:
I already said I fixed it, and your walls of text should be OK, assuming you have strings showing

Reply:
Yes I have FTP access.
but if you look into the directories you will see that for the R7800 mostly no images are built.

If there are no images built for my router then I can't test the build ;-)

Reply:
right, well soon as all is merged I can ask BS to build yours and mine